### PR TITLE
Update Japanese localization on concepts/configuration/configmap.md

### PR DESCRIPTION
**Pull request by hase1128**
**What this PR does / why we need it:**
concepts/configuration/configmap.md is outdated.

File to update
https://github.com/kubernetes/website/tree/dev-1.19-ja.1/content/ja/docs/concepts/configuration/configmap.md

Original
https://github.com/kubernetes/website/tree/release-1.19/content/en/docs/concepts/configuration/configmap.md

diff between translated and v1.19
https://gist.github.com/e92f6d3b819b60295505744afabab6f9

**Which issue(s) this PR fixes:**
#26859

### CODE REVIEW

File: content/ja/docs/concepts/configuration/configmap.md [hunk 1]
```diff
@@ -180,9 +188,7 @@ data:
 immutable: true
 ```
 
-{{< note >}}
-一度ConfigMapやSecretがイミュータブルに設定すると、この変更を元に戻したり、`data`フィールドのコンテンツを変更することは*できません*。既存のPodは削除されたConfigMapのマウントポイントを保持するため、こうしたPodは再作成することをおすすめします。
-{{< /note >}}
+一度ConfigMapがイミュータブルに設定されると、この変更を元に戻したり、`data`または`binaryData`フィールドのコンテンツを変更することは*できません*。Configmapの削除と再作成のみ可能です。既存のPodは削除されたConfigMapのマウントポイントを保持するため、こうしたPodは再作成することをおすすめします。
```
Comment (makocchi-git) [inline content/ja/docs/concepts/configuration/configmap.md:191]: 他は lgtm です
```suggestion
一度ConfigMapがイミュータブルに設定されると、この変更を元に戻したり、`data`または`binaryData`フィールドのコンテンツを変更することは*できません*。ConfigMapの削除と再作成のみ可能です。既存のPodは削除されたConfigMapのマウントポイントを保持するため、こうしたPodは再作成することをおすすめします。
```

**Comment by hase1128** [comment]
@makocchi-git 
ご指摘ありがとうございます。suggestionをaccepctしました。

File: content/ja/docs/concepts/configuration/configmap.md [hunk 2]
```diff
@@ -22,12 +22,18 @@ ConfigMapは機密性や暗号化を提供しません。保存したいデー
 
 こうすることで、必要であればクラウド上で実行しているコンテナイメージを取得することで、ローカルでも完全に同じコードを使ってデバッグができるようになります。
 
+ConfigMapは、大量のデータを保持するようには設計されていません。ConfigMapに保存されるデータは1MiBを超えることはできません。この制限を超える設定を保存する必要がある場合は、ボリュームのマウントを検討するか、別のデータベースまたはファイルサービスを使用することを検討してください。
+
 ## ConfigMapオブジェクト
 
-ConfigMapは、他のオブジェクトが使うための設定を保存できるAPI[オブジェクト](/ja/docs/concepts/overview/working-with-objects/kubernetes-objects/)です。ほとんどのKubernetesオブジェクトに`spec`セクションがあるのとは違い、ConfigMapにはアイテム(キー)と値を保存するための`data`セクションがあります。
+ConfigMapは、他のオブジェクトが使うための設定を保存できるAPI[オブジェクト](/ja/docs/concepts/overview/working-with-objects/kubernetes-objects/)です。ほとんどのKubernetesオブジェクトに`spec`セクションがあるのとは違い、ConfigMapには`data`および`binaryData`フィールドがあります。これらのフィールドは、キーとバリューのペアを値として受け入れます。`data`フィールドと` binaryData`フィールドはどちらもオプションです。`data`フィールドはUTF-8バイトシーケンスを含むように設計されていますが、` binaryData`フィールドはバイナリデータを含むように設計されています。
```
Comment (inductor) [inline content/ja/docs/concepts/configuration/configmap.md:29]:
```suggestion
ConfigMapは、他のオブジェクトが使うための設定を保存できるAPI[オブジェクト](/ja/docs/concepts/overview/working-with-objects/kubernetes-objects/)です。ほとんどのKubernetesオブジェクトに`spec`セクションがあるのとは違い、ConfigMapには`data`および`binaryData`フィールドがあります。これらのフィールドは、キーとバリューのペアを値として受け入れます。`data`フィールドと`binaryData`フィールドはどちらもオプションです。`data`フィールドはUTF-8バイトシーケンスを含むように設計されていますが、`binaryData`フィールドはバイナリデータを含むように設計されています。
```

File: content/ja/docs/concepts/configuration/configmap.md [hunk 3]
```diff
@@ -22,12 +22,18 @@ ConfigMapは機密性や暗号化を提供しません。保存したいデー
...
 
 ConfigMapの名前は、有効な[DNSのサブドメイン名](/ja/docs/concepts/overview/working-with-objects/names#dns-subdomain-names)でなければなりません。
 
+`data`または` binaryData`フィールドの各キーは、英数字、`-`、`_`、または`.`で構成されている必要があります。`data`に格納されているキーは、`binaryData`フィールドのキーと重複することはできません。
```
Comment (inductor) [inline content/ja/docs/concepts/configuration/configmap.md:33]:
```suggestion
`data`または`binaryData`フィールドの各キーは、英数字、`-`、`_`、または`.`で構成されている必要があります。`data`に格納されているキーは、`binaryData`フィールドのキーと重複することはできません。
```

**Review by inductor — CHANGES_REQUESTED** [review]

**Comment by hase1128** [comment]
@inductor 

I accepted your suggestion, thank you.

**Pull request merged**